src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -44,6 +44,13 @@ public sealed class CompletionItem : IComparable<CompletionItem>
         /// </summary>
         public string SortText { get; }
 
+        /// <summary>
+        /// Descriptive text to place after <see cref="DisplayText"/> in the display layer.  Should
+        /// be short as it will show up in the UI.  Display will present this in a way to distinguish
+        /// this from the normal text (for example, by fading out and right-aligning).
+        /// </summary>
+        internal string InlineDescription { get; }
```

Comment:
@dpoeschl i could extract out this part of the change into a separate PR if you want.

Reply:
I'd vote for this -- sounds like the concept isn't finalized with the editor team, and removing the need for the public API review here would be nice for now.

Reply:
Ok.  Will do in the next day or so!